.erb-lint.yml

```diff
@@ -0,0 +1,32 @@
+---
+exclude:
+  - '**/vendor/**/*'
+linters:
+  Rubocop:
+    enabled: true
+    rubocop_config:
+      inherit_from:
+        - .rubocop.yml
+      # Suggested by https://github.com/Shopify/erb-lint#rubocop
+      Layout/InitialIndentation:
+        Enabled: false
+      Layout/LineLength:
+        Enabled: false
+      Layout/TrailingEmptyLines:
+        Enabled: false
+      Style/FrozenStringLiteralComment:
+        Enabled: false
+      Lint/UselessAssignment:
+        Enabled: false
+      Rails/OutputSafety: # Already covered by ErbSafety linter
+        Enabled: false
```

Comment:
I tried removing these but they muck up the autocorrect mechanism. There's a reason the erblint README recommends keeping them off.